x-pack/solutions/observability/plugins/synthetics/server/routes/settings/private_locations/get_agent_policies.ts

```diff
@@ -46,5 +46,6 @@ export const getAgentPoliciesAsInternalUser = async ({
     status: agentPolicy.status,
     description: agentPolicy.description,
     namespace: agentPolicy.namespace,
+    spaceIds: agentPolicy.space_ids || [agentPolicy.namespace],
```

Comment:
agentPolicy.namespace isn't necessarily same as space id.

Reply:
What should be the value of `spaceIds` if `agentPolicy.space_ids` is `undefined`?